arbeitszeit_web/controllers/query_companies_controller.py

```diff
@@ -52,7 +54,7 @@ def import_form_data(
                 filter_category = CompanyFilter.by_email
             else:
                 filter_category = CompanyFilter.by_name
-        offset = self._get_pagination_offset(request) if request else 0
+        offset = self._get_pagination_offset(self.request)
```

Comment:
Since the `Request` object is now an attribute of `QueryCompaniesController` you don't need to pass `self.request` to `self._get_pagination_offset` but can access `self.request` directly in that method.